packages/client/src/containers/Datasets/DatasetDetail.tsx

```diff
@@ -45,6 +45,11 @@ function _DatasetDetail({ getDataset, updateDataset }) {
   const [fileUploaded, setFileUploaded] = React.useState(false);
   const [tabKey, setTabKey] = React.useState('data');
 
+  if (!window?.enablePhfs) {
```

Comment:
You can handle this by the `useEffect`:

```tsx
useEffect(() => {
  if (!window?.enablePhfs) {
    history.push('../home');
  }
}, []);
```

and you don't need to return `<></>`.

Reply:
after discussion with @neighborhood999, we decide not to change